CHANGELOG.md

```diff
@@ -6,6 +6,43 @@ and this project uses `yyyy.rr[.pp]`, where `yyyy` is the year a patch is releas
 `rr` is a sequential release number (starting from `01`), and an optional two-digit
 sequential patch number (starting from `01`).
 
+## [2023.03] - 2023-10-20
+### Known Issues
+- GCC 9 and below as well as GCC 11.1.0 are unsupported due to compilation issues. See prior releases for more details.
+- `NO_QUAD_PRECISION` is no longer set by FMS, the `-DNO_QUAD_PRECISION` flag may need to be added to xml's in order to compile component code
```

Comment:
I'm not sure I consider this a `known issue` as that implies we will may be changing again in the future.  But I'm not really sure how else to make this any more prominent.

Reply:
I can instead (or also) mention it when I send out the email for the release to the user list.

Reply:
With the extra information that users should be using ENABLE_QUAD_PRECISION, let's change the message here to reference the corrected macro.

Reply:
fixed with 5df8fba